### ISSUE TITLE
pytorchcrf.py:305 出现警告信息

### ISSUE THREAD

**Issue by py-rpa** [liwenju0/cutword#21]
Loading dictionaries from D:\ShadowBot\shadowbot-5.17.53\python\lib\site-packages\cutword\dict.txt
D:\ShadowBot\shadowbot-5.17.53\python\lib\site-packages\cutword\ner.py:584: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  return torch.tensor(input_tensors), torch.tensor(seq_lens), input_lists
D:\ShadowBot\shadowbot-5.17.53\python\lib\site-packages\cutword\pytorchcrf.py:305: UserWarning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\TensorCompare.cpp:413.)
  score = torch.where(mask[i].unsqueeze(1), next_score, score)
  
  这个警告信息是正常的吗，环境 python3.7.4  win11   小白不太懂， 劳烦解答

**Comment by liwenju0** [comment]
正常，不用管
